paz/backend/heatmaps.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+
+
+def get_keypoints_heatmap(heatmaps, num_keypoints, indices=None, axis=1):
+    """Extract the heatmaps that only contains the keypoints.
+
+    # Arguments
+        heatmaps: Numpy array.
```

Comment:
Maybe you can tell us the shape of the input and output numpy array i.e. (H, W, 3)

Reply:
done